rollup.config.js

```diff
@@ -29,8 +28,7 @@ const plugins = [
 			['@babel/plugin-transform-computed-properties', { loose: true }],
 			'@babel/plugin-transform-arrow-functions',
 			['@babel/plugin-transform-classes', { loose: true }],
-			['@babel/plugin-proposal-object-rest-spread', { useBuiltIns: true }],
-			'@babel/plugin-external-helpers'
```

Comment:
removed '@babel/plugin-external-helpers'as it was deprecated and not needed anymore